.circleci/README.md

```diff
@@ -46,7 +46,7 @@ You should get a JSON response with a list of country codes.
 
 ### Creating the Fly Configuration file
 
-Next, you need to reserve a namespace on the Fly platform remotely and create a fly configuration file. You need the `flyctl` command line tool for that. If you haven't already, visit [flyctl](https://fly.io/docs/hands-on/installing/) to learn how to install the fly CLI for your machine.
+Next, you need to reserve a namespace on the Fly platform remotely and create a fly configuration file. You need the `flyctl` command-line tool for that. If you haven't already, visit [flyctl](https://fly.io/docs/hands-on/installing/) to learn how to install the fly CLI for your machine.
```

Comment:
Next, you need to initialize an app of Fly. This reserves you an application name and creates a fly configuration file.

(Simpler mapping to the actual command; and it's a name in a namespace thats being reserved)